app/views/reports/templates/template_1/_footer.html.erb

```diff
@@ -0,0 +1,93 @@
+<!doctype html>
+<html>
+<head>
+  <meta charset='utf-8' />
+  <script>
+    function number_pages() {
+      var vars={};
+      var x=document.location.search.substring(1).split('&');
+      for(var i in x) {var z=x[i].split('=',2);vars[z[0]] = decodeURIComponent(z[1]);}
+      var x=['frompage','topage','page','webpage','section','subsection','subsubsection'];
+      for(var i in x) {
+        var y = document.getElementsByClassName(x[i]);
+        for(var j=0; j<y.length; ++j) y[j].textContent = vars[x[i]];
+      }
+    }
+  </script>
+  <style>
+    body {
+      height: 95px;
+    }
+
+    table {
+      border-collapse: collapse;
+      border-spacing: 0;
+      height: 80px;
+      table-layout: fixed;
+      width: 100%
+    }
+
+    table td {
+      border: 2px solid black;
+      font-size: 12px;
+      padding: .25em;
+      text-align: left;
+      vertical-align: top;
+    }
+
+    table td.pagination {
+      font-size: 14px;
+      font-weight: bold;
+      text-align: center;
+      vertical-align: middle;
+      white-space: nowrap;
+    }
+
+    .info-row {
+      height: 15px;
+    }
+
+    .info-row td {
+      font-weight: bold;
+      text-align: center;
+      vertical-align: middle;
+    }
+  </style>
+</head>
+<body onload="number_pages()">
+  <table>
+    <tr>
+      <td colspan=3>
+        <div class="cell-label">
+          <b>ANALYST(S)</b>
+        </div>
+        <div class="cell-value">
+          <%= template[:analysts] %>
+        </div>
+      </td>
+      <td colspan=2>
+        <div class="cell-label">
+          <b>ANALYST EMPLOYEE NO.</b>
+        </div>
+        <div class="cell-value">
+          <%= template[:analyst_number] %>
+        </div>
+      </td>
+      <td colspan=3>
+        <div class="cell-label">
+          <b>CHECKED BY:</b>
+        </div>
+        <div class="cell-value">
+          <%= template[:checked_by] %>
+        </div>
+      </td>
+      <td colspan=2 class="pagination">
+        PAGE <span class="page"></span> OF <span class="topage"></span>
```

Comment:
So [page] doesn't work in html?

Reply:
No, I think it is only for text.

Reply:
So it can be moved out to separate JS file?